### DISCUSSION TITLE
@nuxt/ui - Where can I find a ref in the doc?

### DISCUSSION THREAD

**Discussion by realtebo** [nuxt/nuxt#17666] [General]
I just discovered existance of @nuxt/ui package. 

Is there a some sort of list of official pagackes related tu nuxt3?

**Comment by hslee2008** [comment]
Check out [Modules for V3](https://modules.nuxtjs.org/?version=3.x) in modules.nuxtjs.org.

Use list of modules here for your project (created by the Nuxt team and community).
